setup.py

```diff
@@ -23,25 +24,36 @@ def get_version(verbose=1, filename='src/qtt/version.py'):
     return version
 
 
-extras = {
-    # name: (module_name, minversion, pip_name)
-    'scikit-image': ('skimage', '0.11', 'scikit-image'),
-    'pandas': ('pandas', '0.15', None),
-    'attrs': ('attr', '16.2.0', 'attrs'),
-    'h5py': ('h5py', '0.1', None),
-    'pyzmqrpc': ('zmqrpc', '1.5', None),
-    'pytables': ('tables', '3.2', None),
-    'apscheduler': ('apscheduler', '3.4', None),
-    'Polygon3': ('Polygon', '0.1', None),
-    'pyqt5': ('PyQt5', '0.11', 'pyqt5'),
-}
+tests_require = ['coverage', 'jupyter', 'mypy', 'pytest']
+
+install_requires = [
+    'apscheduler', 'attrs', 'dulwich', 'h5py', 'hickle', 'IPython>=0.1', 'lmfit', 'matplotlib>=3.0',
+    'numpy>=1.15', 'opencv-python', 'PyQt5', 'pyqtgraph', 'pyvisa', 'pyzmqrpc', 'qcodes>=0.5.0',
+    'qilib', 'qtpy', 'qupulse', 'redis', 'scipy>=0.18', 'scikit-image', 'jupyter', 'coverage', 'sympy'
+] + tests_require
 
 if platform.system() == 'Windows':
-    extras['pywin32'] = ('win32', '0.1', None)
+    install_requires.append('pywin32')
+
+if platform.python_version() < "3.7.0":
+    install_requires.append('Polygon3')
+else:
+    if platform.system() == 'Windows':
+        # When Polygon3 not yet in git-repository get it locally
+        file_path = abspath(getsourcefile(lambda: 0))
+        bin_dir = os.path.join(os.path.dirname(file_path), 'bin')
+        Polygon3_local = f'Polygon3 @ file://{bin_dir}/Polygon3-3.0.8-cp37-cp37m-win_amd64.whl'
+        Polygon3_git = 'Polygon3 @ https://github.com/QuTech-Delft/qtt/bin/Polygon3-3.0.8-cp37-cp37m-win_amd64.whl'
```

Comment:
This does not work for me on both Windows. The local wheel does work for me. Should we just use that for now?